### PR TITLE
IT-5983 outline item slot label

### PR DESCRIPTION
**Pull request by pioziol123**

### CODE REVIEW

File: lib/js/components/Outline/OutlineItem/OutlineItem.vue [hunk 1]
```diff
@@ -26,8 +26,15 @@
 				:icon="iconLeft"
 				:size="ICON_SIZES.X_SMALL"
 			/>
-			<span class="ds-outlineItem__text">
+			<span v-else class="ds-outlineItem__text">
```
Comment (kvas-damian) [inline lib/js/components/Outline/OutlineItem/OutlineItem.vue:29]: why `v-else`?

Isn't it icon + label a valid use-case?

File: lib/js/components/Outline/OutlineItem/OutlineItem.vue [hunk 2]
```diff
@@ -26,8 +26,15 @@
...
 				<span class="ds-outlineItem__label" :class="{ '-ds-uppercase': isLabelUppercase }">
+					<template v-if="$slots.label">
+						<slot name="label" />
```
Comment (kvas-damian) [inline lib/js/components/Outline/OutlineItem/OutlineItem.vue:32]: should we mark it in a storybook that this component accept both prop and slot?

File: lib/js/components/Outline/OutlineItem/OutlineItem.vue [hunk 3]
```diff
@@ -84,6 +91,7 @@
 		align-items: flex-start;
 		column-gap: $space-2xs;
 		display: flex;
+		max-width: 100%;
```
Comment (kvas-damian) [inline lib/js/components/Outline/OutlineItem/OutlineItem.vue:94]: what does it change?
When I put a long label without spaces, then text goes outside this component

Comment (pioziol123) [reply lib/js/components/Outline/OutlineItem/OutlineItem.vue:94]: Without it overflow does not work on .katex and .katex-display classes

File: lib/js/components/Outline/OutlineItem/OutlineItem.vue (hunk 2 above)
Comment (pioziol123) [reply lib/js/components/Outline/OutlineItem/OutlineItem.vue:32]: yeah, u right 😒

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**